tools/test/integration_test.py

```diff
@@ -192,3 +200,12 @@ def test_bad_click_host(self, settings):
         assert len(tiles) == 2
         names = map(lambda tile: tile.get("name").lower(), tiles)
         assert list(names) == ["acme", "dunder mifflin"]
+
+    def test_aatimeout(self, settings):
+        # restart the test server with a timeout response
+        global SERVER
+        kill_process(SERVER)
+        setup_module(test_mode="TestTimeout")
+        url = "{root}/v1/tiles".format(root=settings.get("test_url"))
+        resp = requests.get(url, headers=default_headers(test=""))
+        assert resp.status_code == 204
```

Comment:
With `AdmLoadError` going through the error handling path we might as well also `assert not resp.content`. I think `actix-web` doesn't render anything for 204s despite our error handler setting a JSON response body but let's test it.